hazelcast-client-legacy/src/test/java/com/hazelcast/client/ClientServiceTest.java

```diff
@@ -144,7 +148,23 @@ public void testNumberOfClients_afterUnAuthenticatedClient_withTwoNode() {
         assertEquals(0, instance2.getClientService().getConnectedClients().size());
     }
 
-
+    @Test
+    public void loadServiceHooksClassNames() throws IOException {
```

Comment:
+1

if you are doing copy paste, something is going wrong. Copy paste is violation or our architectural principles.